src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -149,4 +151,18 @@ class GoogleExtensionSpec extends FlatSpec with Matchers with TestSupport with M
     val petSaResponse2 = googleExtensions.createUserPetServiceAccount(defaultUser).futureValue
     petSaResponse2 shouldBe emailResponse
   }
+
+  it should "get a group's last synchronized date" in {
+    val groupName = WorkbenchGroupName("group1")
+
+    val mockDirectoryDAO = mock[DirectoryDAO]
```

Comment:
It appears we don't currently have a test that actually performs group sync against real Google/LDAP (unless maybe it's in Ursa's PR?). That test would make this more superficial test more meaningful and also provide a skeleton for writing a more meaningful test for this endpoint.

Reply:
we don't have anything against a real google, that should be in the up coming api integration tests